.github/workflows/test.yaml

```diff
@@ -63,4 +66,20 @@ jobs:
 
       - run: patchright install --no-shell chromium
 
-      - run: pytest tests/${{ matrix.test }}.py
+      - name: Install Playwright and Chromium
+        run: |
+          uv pip install playwright
+          uv run playwright install chromium
+          uv run patchright install chrome
+
+      - name: Launch Chrome with remote debugging
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y google-chrome-stable
+          google-chrome --headless --disable-gpu --remote-debugging-port=9222 --no-sandbox &
+
+          # Give Chrome a few seconds to start
+          sleep 5
```

Comment:
no need, remove all of this, the test runners already handle chromium install perfectly and chrome startup time is <1s so it's fine ot launch one for each test file